### PR TITLE
disable AI use of camera computers

### PR DESCRIPTION
**Pull request by toomykins**
[tweak]
Currently only works in a buggy fashion, leaves actionbuttons on the screen that cant be gotten rid of, and becomes even buggier upon a client restart, not allowing you to switch to different cameras with the actionbuttons and always bringing your field of view back to the AI core.
Seems like a hassle to fix, /mob/living/silicon/ai/cancel_camera() needs a work-over, obj/machinery/computer/security/proc/set_camera should make use of /mob/living/silicon/ai/proc/switchCamera and checks all over camera.dm seem to forgo taking AI interaction into account.

AI has built-in systems. Until fixed, this should be disabled.
I don't think any functionality will be lost with this, since AI has access to all cameras anyway, in a better way, too.

:cl:
 * tweak: disabled AI use of camera computers

### CODE REVIEW

**Comment by toomykins** [comment]
this compiles, travis just fucked up a HTTP request

**Comment by boy2mantwicethefam** [comment]
Might want to have it check if the user is a borg instead unless they already got that interaction

**Comment by toomykins** [comment]
borgs and mommers should now be able to use the camera pc's if they stand next to them

File: code/game/machinery/computer/camera.dm [hunk 1]
```diff
@@ -51,8 +51,15 @@ var/list/obj/machinery/camera/cyborg_cams = list(
 	..()
 
 /obj/machinery/computer/security/attack_ai(var/mob/user)
-	src.add_hiddenprint(user)
-	return attack_hand(user)
+	if(istype(user, /mob/living/silicon/robot) || isMoMMI(user))
```
Comment (jknpj) [inline code/game/machinery/computer/camera.dm:54]: borgs and mommis don't call attack_ai at melee range, they call attack_robot()...

File: code/game/machinery/computer/camera.dm [hunk 2]
```diff
@@ -51,8 +51,15 @@ var/list/obj/machinery/camera/cyborg_cams = list(
...
+		if(Adjacent(user))
+			src.add_hiddenprint(user)
+			return attack_hand(user)
+		else
+			to_chat(user, "You need to get closer to the computer first.")
+	else
+		to_chat(user, "You have your built-in camera systems for this!") //currently too buggy to allow AI to use camera computers
+	return //attack_hand(user)
```
Comment (jknpj) [inline code/game/machinery/computer/camera.dm:62]: useless return

File: code/game/machinery/computer/camera.dm (hunk 1 above)
Comment (jknpj) [reply code/game/machinery/computer/camera.dm:54]: I'm dumb

**Pull request merged**